### DISCUSSION TITLE
Any plans to support rotation angle?

### DISCUSSION THREAD

**Discussion by jonhaber82** [JeffSteinbok/hass-dreo#133] [Ideas]
I am curious if you plan to support or already do support this functionality (oscillation angle)?

![Screenshot_20240717-124506](https://github.com/user-attachments/assets/fa933e2a-e265-472b-a57d-48ee3bd62c05)

Thanks for making this great integration, it's very useful!

**Comment by JeffSteinbok** [comment]
Please file an issue if not already there. List it under new device support and include logs.
